exclusions.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
In retrospect, I totally see your point: We could really just _remove_ these from `hub.json`, rather than having a totally separate `exclusions.json` to muddy the waters. Apologies for having made the additional ask; no need to make any changes for now.

Reply:
All good. And I have this in my dev notes to eventually take care of this but frankly, the reason I'm not is that this will ultimately require telling people their packages have been delisted from hub.json (after they thought they had been added). To make this a smooth conversation, we need to provide a good contributing spec, which is something I believe I'll be tackling later this quarter as we think about revamping documentation and contribution guidelines.